AutoGear.lua

```diff
@@ -4248,10 +4252,20 @@ function AutoGearTooltipHook(tooltip)
 		)
 	end
 end
-GameTooltip:HookScript("OnTooltipSetItem", AutoGearTooltipHook)
-ShoppingTooltip1:HookScript("OnTooltipSetItem", AutoGearTooltipHook)
-ShoppingTooltip2:HookScript("OnTooltipSetItem", AutoGearTooltipHook)
-ItemRefTooltip:HookScript("OnTooltipSetItem", AutoGearTooltipHook)
+
+if TOC_VERSION_CURRENT < TOC_VERSION_DF then
+	GameTooltip:HookScript("OnTooltipSetItem", AutoGearTooltipHook)
+	ShoppingTooltip1:HookScript("OnTooltipSetItem", AutoGearTooltipHook)
+	ShoppingTooltip2:HookScript("OnTooltipSetItem", AutoGearTooltipHook)
+	ItemRefTooltip:HookScript("OnTooltipSetItem", AutoGearTooltipHook)
+else
+	-- For Dragonflight and beyond, registering callbacks with the new TooltipDataProcessor.AddTooltipPostCall function is the way to hook tooltips.
+	-- Source: https://wowpedia.fandom.com/wiki/Patch_10.0.2/API_changes#Tooltip_Changes
+	TooltipDataProcessor.AddTooltipPostCall(GameTooltip, AutoGearTooltipHook)
+	TooltipDataProcessor.AddTooltipPostCall(ShoppingTooltip1, AutoGearTooltipHook)
+	TooltipDataProcessor.AddTooltipPostCall(ShoppingTooltip2, AutoGearTooltipHook)
+	TooltipDataProcessor.AddTooltipPostCall(ItemRefTooltip, AutoGearTooltipHook)
```

Comment:
Still working on this one

It isn't showing the Tooltip but it isn't breaking the client either. You are able to still use the addon in Dragonflight and SoD/Cata

Reply:
That looks very familiar.  I think that exact code is part of what I had tried when I last worked on this, but it didn't work.

Reply:
-- changed in Dragonflight 10.2.6
-- source: https://warcraft.wiki.gg/wiki/API_C_Item.GetItemInfo
-- source: https://warcraft.wiki.gg/wiki/API_C_Item.IsEquippedItem

Changes for GetItemInfo and IsEquippedItem as well

Reply:
Since this change does not resolve the tooltip issue, I think these Dragonflight-specific fixes should be removed from this pull request and separated into a different pull request.  That would require having the changes in a different branch in your fork.

Reply:
took the dragonflight stuff out